server/src/server.ts

```diff
@@ -50,7 +50,7 @@ connection.onInitialize(
             capabilities: {
                 // Tell the client that the server support code complete
                 completionProvider: {
-                    resolveProvider: true,
+                    triggerCharacters: ['.', ':'],
```

Comment:
I guess it would have a poor “simplicity to usefulness” ratio, but I don't like the fact that if you

1) type two colons and
2) use backspace to erase one

the suggestions disappear and you have to hit Ctrl-Space yourself xd

Reply:
What trigger characters should I set?

Reply:
I dunno, I was just pointing at the fact that this system wasn't perfect for our case, but I doubt it would be easy to fix that, nor that it would be worth the trouble.

Reply:
I will keep the dot alone, then.

Reply:
Wut? No! The colon is helpful, dude. What are you talking about? xD